jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/redis/RedisConfiguration.java

```diff
@@ -14,20 +16,20 @@
 import org.springframework.data.redis.connection.RedisSentinelConfiguration;
 import org.springframework.data.redis.connection.RedisStandaloneConfiguration;
 import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
+import org.springframework.integration.redis.metadata.RedisMetadataStore;
 
 import javax.naming.OperationNotSupportedException;
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
 
-
 /**
  * Redis configuration properties
  * @author alexjoybc
  * @since 0.4.0
  */
 @Configuration
-@ConditionalOnExpression("'${bcgov.access.input.plugin}' == 'rabbitmq' || '${bcgov.access.output.plugin}' == 'rabbitmq'")
+@ComponentScan
 public class RedisConfiguration {
```

Comment:
for consistency we could rename it to `AutoConfiguration`

Reply:
agree.
Changed.